hoodie-client/src/test/java/com/uber/hoodie/index/bloom/TestIndexLookUpTree.java

```diff
@@ -0,0 +1,191 @@
+/*
+ *  Copyright (c) 2018 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package com.uber.hoodie.index.bloom;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Random;
+import org.junit.Test;
+
+/**
+ * Tests {@link IndexLookUpTree}
+ */
+public class TestIndexLookUpTree {
+
+  public static final Random RANDOM = new Random();
+  private IndexLookUpTree indexLookUpTree;
+  private Map<String, HashSet<String>> expectedMatches;
+
+  public TestIndexLookUpTree() {
+    indexLookUpTree = new IndexLookUpTree();
+    expectedMatches = new HashMap<>();
+  }
+
+  /**
+   * Generates and returns a random string of given length
+   *
+   * @param length the length of the random string to be generated
+   * @return the random string thus generated
+   */
+  private static String getRandomString(int length) {
+    String salChars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";
```

Comment:
can you test with `UUID.randomUUID` instead? thats how the typical key/filenames are generated in tests